group/views.py

```diff
@@ -50,11 +50,10 @@
 def get_running_contest(request, group_id):
 
     group = get_group(group_id)
-
-    all_running_contest_list = []
     now = timezone.now()
 
     all_running_contest_list = group.trace_contest.filter(start_time__lte=now, end_time__gte=now)
+    all_running_contest_list = get_current_page(request, all_running_contest_list)
```

Comment:
I guess this line is for pager, right?
Can you use other variable name to avoid confusing?